roles/pulp-webserver/templates/nginx.conf.j2

```diff
@@ -84,8 +84,15 @@ http {
         include conf.d/*;
 
         location / {
-            # checks for static file, if not found proxy to app
-            try_files $uri =404;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+            proxy_set_header Host $http_host;
+            # we don't want nginx trying to do something clever with
+            # redirects, we set the Host: header above already.
+            proxy_redirect off;
+            proxy_pass http://pulp-api;
+
+            proxy_next_upstream http_404;  # Try the other service upon 404
```

Comment:
I think we should drop this line because we can't maintain consistency with apache. I spent a while looking into how to make it consistent and I determined it's not possible.